txpool/txpool.go

```diff
@@ -589,6 +599,66 @@ func (t *TxPool) decreaseSlots(slots uint64) {
 	atomic.AddUint64(&t.slots, ^(slots - 1))
 }
 
+func (t *TxPool) isSpeedUp(tx *types.Transaction) (*types.Transaction, bool) {
+	nextNonce, ok := t.GetNonce(tx.From)
+	if !ok {
+		println("No previous txs for this acc")
+		return nil, false
+	}
+
+	mux := t.lockAccountQueue(tx.From, false)
+	defer mux.unlock()
+
+	if nextNonce == tx.Nonce+1 {
+		// old tx was promoted to pending queue
+		oldTx := mux.accountQueue.lastPromoted
+		return oldTx, true
+	} else if nextNonce < tx.Nonce {
+		// old tx might be waiting on promotion (if it's there)
+		i := sort.Search(mux.accountQueue.txs.Len(), func(i int) bool {
+			return mux.accountQueue.txs[i].Nonce >= tx.Nonce
+		})
+
+		if i < mux.accountQueue.txs.Len() && mux.accountQueue.txs[i].Nonce == tx.Nonce {
+			// found it
+			return mux.accountQueue.txs[i], true
+		}
+	}
+
+	return nil, false
+}
+
+func (t *TxPool) speedUp(newTx, oldTx *types.Transaction) error {
```

Comment:
please document this method

Reply:
[Documented](https://github.com/0xPolygon/polygon-sdk/pull/220/commits/8e4ce12c2c344c912148fb5a04a2c386774a0836)